ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/push-queries.json

```diff
@@ -653,6 +653,26 @@
         ]}
       ]
     },
+    {
+      "name": "aggregate without GROUP BY expression",
```

Comment:
can we add tests for (1) what happens if you try to `CREATE TABLE` without a group by (basically confirm a good error message), (2) what happens if you select a non-grouping column (e.g. `SELECT COUNT(*), ID`) and (3) that multiple aggregate functions work?

Reply:
I will add tests covering cases 1 and 3.

For 2, this is what currently happens 
```
ksql> select avg_stars,count(*) from RATINGS_TABLE EMIT CHANGES;
Non-aggregate SELECT expression(s) not part of GROUP BY: AVG_STARS
Either add the column(s) to the GROUP BY or remove them from the SELECT.
```
Is that reasonable?

Reply:
discussed offline, confirmed this was reasonable

Reply:
How about a stream pull query case, like "SELECT count(column) AS count FROM test"?

Reply:
I have manually tested the stream pull query case, but I can also add an explicit test case for it too

Reply:
Actually pull queries dont support group by clauses at all, unless you mean add a test to make sure we get the expected error? This is the current error
```
Pull queries don't support GROUP BY clauses. See https://cnfl.io/queries for more info.
Add EMIT CHANGES if you intended to issue a push query.
Statement: select count(*), sum(user_id) from RATINGS_STREAM;
```